codec.proto

```diff
@@ -60,21 +66,22 @@ message BlockParams {
 }
 
 message Commit {
-  int64               height      = 1; // must be > 0
-  int32               round       = 2; // must be > 0
-  BlockID             block_id    = 3 [(gogoproto.nullable) = false, (gogoproto.customname) = "BlockID"];
-  repeated CommitSig  signatures  = 4 [(gogoproto.nullable) = false];
+  int64              height     = 1; // must be > 0
+  int32              round      = 2; // must be > 0
+  BlockID            block_id   = 3 [(gogoproto.nullable) = false, (gogoproto.customname) = "BlockID"];
+  repeated CommitSig signatures = 4 [(gogoproto.nullable) = false];
 }
 
 message CommitSig {
   BlockIDFlag block_id_flag     = 1; // must match those in the enum above
-  bytes       validator_address = 2; // hash 256 20 bytes - this is the first 20 characters of a 32-byte key - SHA256(pubkey)[:20]
-  Timestamp   timestamp         = 3 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
-  bytes       signature         = 4; // length should be > 0 and < 64
+  bytes       validator_address = 2; // hash 256 20 bytes - this is the first 20 characters of a 32-byte key -
+                                     // SHA256(pubkey)[:20]
```

Comment:
Comments broken into multiple lines do not translate well to the generated files ([example](https://github.com/figment-networks/graph-node/commit/e26516e5eb5cbbe8a9f636788bee77b5e7442947#diff-25193c25bf9f48bedbed65facbcd45bbd38a39b01373bee892b4c4be1176bda0R64-R67)). Maybe we can tweak the formatter (or make the comment shorter)?